desktop/src/main/java/bisq/desktop/primary/main/content/trade/overview/TradeOverviewModel.java

```diff
@@ -29,4 +33,93 @@ public class TradeOverviewModel implements Model {
     final ObservableList<ProtocolListItem> listItems = FXCollections.observableArrayList();
     final FilteredList<ProtocolListItem> filteredItems = new FilteredList<>(listItems);
     final SortedList<ProtocolListItem> sortedItems = new SortedList<>(filteredItems);
+    
+    public void initialize() {
```

Comment:
Main reason I moved it here was to avoid duplication in list/grid controller. Do you think's it's okay to introduce service layer with method that returns list of available protocols? In which package if yes (common, desktop)?

Reply:
I think its best here to make full inheritence for all the MCV classes. So there are 2 controllers extending a basecontroller, same for models and views. Its a bit boilerplate but keeps structures clean, otherwise shortterm shortcuts lead to more messy structure later and over time more stuff gets in and then the initial boilerplate pays off.

Reply:
Services should only be used for real domain data, and then they are in the relevant domains (e.g. social). For pure UI state we should keep things in the models.

Reply:
[Fixed](https://github.com/bisq-network/bisq2/pull/297/commits/97935a986e6bd47ea3f7efcb89463b0573d7802b) - full inheritence for all the MCV classes introduced